### DISCUSSION TITLE
How to load the custom font in Weblate?

### DISCUSSION THREAD

**Discussion by Habi-Thapa** [WeblateOrg/weblate#5451] [Q&A]
I uploaded my custom font named "Preeti" for the Devanagari script and override the Nepali language and Hindi language from the Weblate `Manage > Fonts > Font Group` section. Yet I am unable to load that font in my Weblate project. Do I need to update anything else?

**Comment by nijel** [comment]
Using the fonts for the maximal size check is described in the documentation: https://docs.weblate.org/en/latest/admin/checks.html#managing-fonts

**Reply by Habi-Thapa** [reply]
I just saw that update in the documentation. Being able to read those fonts in Weblate would also be helpful. We could have used unicode-fonts instead of "Preeti" font but some letters do not render properly in Unicode.    

Anyway, thanks :)
